stage1/rootfs/usr_from_coreos/cache.sh

```diff
@@ -165,8 +165,8 @@ function cache_url() {
 	if ! gpg_verify "${cache}" "${sigfile}" "${key}" "${keyid}"; then
 
 		# refresh the cache on failure, and verify it again
-		curl --location --output "${cache}" "${url}"
-		curl --location --output "${sigfile}" "${sigurl}"
+		wget -t 20 -O "${cache}" "${url}" # the wget default for retries is 20 times.
```

Comment:
can you use long flags please?
Why are you specifying --tries as the default value?